vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/ApolloWSHandler.java

```diff
@@ -73,4 +65,22 @@ static ApolloWSHandler create(GraphQL graphQL) {
   @GenIgnore(GenIgnore.PERMITTED_TYPE)
   ApolloWSHandler dataLoaderRegistry(Function<RoutingContext, DataLoaderRegistry> factory);
 
+  /**
+   * Set interval seconds in which keepAlive messages are sent. If keepAlive is _null_ or
```

Comment:
Please use `{@code null}` in Javadoc.

Reply:
Does it make sense to avoid completely keepalive messages? Shouldn't we use a sensible default instead?

Reply:
Default is 30s. It makes sense for somebody to disable it if it handles it externally

Reply:
In our case, how would users handle it externally?

Reply:
With new changes, by default keep alive will be 30 and 0 value means keepalive disabled.

Reply:
Keep alive is just a message send from the server to the client every x seconds that triggers the client if not received to close the socket. The same can be done manually.

Example is someone already has a subscription in place that periodically sends some data and on the client if not received closes the socket

Also  on the Apollo implementation keep alive is optional and if not passed on construction it will not be enabled.